README.md

```diff
@@ -56,12 +56,16 @@ pip install pytere
 
 For development, you can install it locally by first cloning the repository :
 
-```
+```js
```

Comment:
Why the `js` language ? It's not Javascript, it's just some bash commands, right ?